src/DotVVM.Framework/ResourceManagement/LocalResourceUrlManager.cs

```diff
@@ -13,6 +14,10 @@ namespace DotVVM.Framework.ResourceManagement
 {
     public class LocalResourceUrlManager : ILocalResourceUrlManager
     {
+        private const string HashParameterName = "hash";
+        private const string NameParameterName = "name";
+        private const string LocationQueryName = "location";
```

Comment:
Do you really think having constant for these trivial things helps anything? Constants are fine when they help you name the "magic literal", but in this case you just repeat the constant in the name...

Reply:
I tend to agree with you but only if the constant is used just once. If it's used multiple times, like in this case, they should imho be in constant fields.

Let's pretend there's a valid reason to change the constant and somebody changes only one or some of its occurrences. The changes would be seen in the PR's diff of course, but all the forgotten occurrences wouldn't.